ledger/sandbox/BUILD.bazel

```diff
@@ -411,36 +409,10 @@ server_conformance_test(
     servers = SERVERS,
     test_tool_args = [
         "--open-world",
-        "--all-tests",
-        "--exclude=ConfigManagementServiceIT",
         "--exclude=ClosedWorldIT",
     ],
 )
 
-server_conformance_test(
-    name = "conformance-test-config-management",
-    server_args = [
-        "--wall-clock-time",
-    ],
-    servers = SERVERS,
-    test_tool_args = [
-        "--open-world",
-        "--include=ConfigManagementServiceIT",
-    ],
-)
-
-server_conformance_test(
-    name = "conformance-test-command-deduplication",
```

Comment:
We're removing it because it doesn't need to be run separately anymore?

Reply:
Exactly. Less CI time to start up a sandbox just for one test. :slightly_smiling_face: 

(kudos to @dasormeter for making me think harder about this one)